### ISSUE TITLE
New SimpleHiScores attack: Modified Carlini & Wagner L2 attack

### ISSUE THREAD

**Issue by dijksterhuis** [dijksterhuis/cleverspeech-exp#6]
Take the `f6` attack and modify it to calculate `kappa` per alignment frame?

The alignment we want only needs to be `>0.5` in the `softmax` for each element. The current `next softmax` could be calculated on the fly from the existing `other logits`.

Any other alignment combination shouldn't be a problem if this is successful `=>` the most likely character per frame (our target characters) will always be more likely than the next likely characters.
